examples/bollinger/bin/download_stock_data.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../bollinger'))
+
+import bollinger.lib as bol
+
+path = bol.normalize_path("all_stocks_5yr.csv")
```

Comment:
Thoughts on making this the first asset in the graph?

Reply:
It already is (in a sense, it first checks for local copy and will download if not found). I made a bunch of small changes like this before posting the PR.